package-lock.json

```diff
@@ -1199,18 +1199,6 @@
         "js-yaml": "bin/js-yaml.js"
       }
     },
-    "node_modules/@eslint/eslintrc/node_modules/minimatch": {
```

Comment:
How are these changes related? Do we have any dependency updates?

Reply:
They are not, but I did run an audit and fix as we had critical vulnerabilities.

Reply:
Ah alright. Let's mark this in an additional commit.